docs/content/tutorial/custom-steps.md

```diff
@@ -0,0 +1,154 @@
+4. Run the following commands in your terminal to store your environment variables, client ID, and client secret.
+5. You'll also need to know your team ID (found by opening your Slack instance in a web browser and copying the value within the link that starts with the letter **T**) and your app ID (found under **Basic Information**).
+
+**For macOS**
+```bash
+export SLACK_BOT_TOKEN=<your-bot-token>
+export SLACK_APP_TOKEN=<your-app-token>
+export JIRA_CLIENT_ID=<client-id>
+export JIRA_CLIENT_SECRET=<client-secret>
+```
+
+**For Windows**
+```bash
+set SLACK_BOT_TOKEN=<your-bot-token>
+set SLACK_APP_TOKEN=<your-app-token>
+set JIRA_CLIENT_ID=<client-id>
+set JIRA_CLIENT_SECRET=<client-secret>
+```
+
+## Setting up and running your local project {#configure-project}
+
+Clone the starter template onto your machine by running the following command:
+
+```bash
+git clone https://github.com/slack-samples/bolt-python-jira-functions.git
+```
+
+Change into the new project directory:
+
+```bash
+cd bolt-python-jira-functions
+```
+
+Start your Python virtual environment:
+
+**For macOS**
+```bash
+python3 -m venv .venv
+source .venv/bin/activate
+```
+
+**For Windows**
+```bash
+py -m venv .venv
+.venv\Scripts\activate
+```
+
+Install the required dependencies:
+
+```bash
+pip install -r requirements.txt
+```
+
+Rename the `.example.env` file to `.env` and replace the values for each of the variables listed in the file:
+
+```
+JIRA_BASE_URL=https://your-jira-instance.com
+SECRET_HEADER_KEY=Your-Header
+SECRET_HEADER_VALUE=abc123
+JIRA_CLIENT_ID=abc123
+JIRA_CLIENT_SECRET=abc123
+APP_BASE_URL=https://1234-123-123-12.ngrok-free.app
+APP_HOME_PAGE_URL=slack://app?team=YOUR_TEAM_ID&id=YOUR_APP_ID&tab=home
+```
+
+Start your local server:
+
+```bash
+python app.py
+```
+
+If your app is up and running, you'll see a message noting that the app is starting to receive messages from a new connection.
+
+## Setting up your workflow in Workflow Builder {#workflow}
+
+1. Within your development workspace, open Workflow Builder by clicking your workspace name and then selecting **Tools** > **Workflow Builder**. 
+2. Select **New Workflow** > **Build Workflow**.
+3. Click **Untitled Workflow** at the top of the pane to rename your workflow. We'll call it **Create Issue**. For the description, enter _Creates a new issue_, then click **Save**.
+
+![Workflow details](/img/custom-steps-jira/1.png)
+
+4. Select **Choose an event** under **Start the workflow...**, and then select **From a link in Slack**. Click **Continue**.
+
+![Start the workflow](/img/custom-steps-jira/2.png)
+
+5. Under **Then, do these things** click **Add steps** to add the custom step. Scroll down to the bottom of the list on the right-hand pane and select **Custom**, then **BoltPy Jira Functions** > **Create an issue**. Enter the project details, issue type (optional), summary (optional), and description (optional). Click **Save**.
```

Comment:
I think it might be helpful to mention that the custom step is what is defined in the https://github.com/slack-samples/bolt-python-jira-functions/blob/main/listeners/functions/create_issue.py file